replay/session_handler.py

```diff
@@ -100,9 +100,11 @@ def __init__(
         device: Optional[torch.device] = None,
     ):
         Borg.__init__(self)
-        if not hasattr(self, "logger_set"):
-            self.logger = logger_with_settings()
-            self.logger_set = True
+        # Old logger logic.
```

Comment:
Don't leave it like a comment, just remove it.
We have history to refer to if needed.